packages/fxa-payments-server/src/components/fields.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useContext, useCallback, useRef, useEffect, DetailedHTMLProps, FormHTMLAttributes } from 'react';
```

Comment:
This module is all fairly light wrappers for HTML input elements with added validation support.